public/app/features/dashboard-scene/serialization/transformSceneToSaveModelSchemaV2.ts

```diff
@@ -367,6 +387,64 @@ function createElements(panels: PanelKind[]): Record<string, PanelKind> {
   );
 }
 
+function panelRepeaterToPanels(repeater: DashboardGridItem, isSnapshot = false): GridLayoutItemKind[] {
```

Comment:
This function's name is confusing, we could change it to  something like `transformRepeatedGridItemToLayoutItemKind` or `repeaterToLayoutItems`